Source/AlertController.swift

```diff
@@ -324,6 +325,9 @@ public final class AlertController: UIViewController {
         }
 
         self.alert.layoutIfNeeded()
+
+        self.view.accessibilityIdentifier = type(of: self).description()
+        self.alert.accessibilityIdentifier = type(of: self.alert).description()
```

Comment:
These changes seem irrelevant. If they represent some other type of bug, can you please open a separate ticket?